src/sbt-test/sbt-scalafix/local-rules/build.sbt

```diff
@@ -10,28 +10,28 @@ inThisBuild(
     // error: Static methods in interface require -target:jvm-1.8
     scalafixResolvers := List(
       coursierapi.MavenRepository.of("https://repo1.maven.org/maven2")
-    )
+    ),
+    scalaVersion := "2.13.0", // out of sync with scalafix.sbt.BuildInfo.scala213 on purpose
+    scalafixScalaBinaryVersion :=
+      // this should be the default in sbt-scalafix 1.0
+      CrossVersion.binaryScalaVersion(scalaVersion.value)
   )
 )
 
 val rules = project
   .disablePlugins(ScalafixPlugin)
   .settings(
-    scalaVersion := Versions.scala212,
-    crossPaths := false,
     libraryDependencies += "ch.epfl.scala" %% "scalafix-core" % Versions.scalafixVersion,
     libraryDependencies += "joda-time" % "joda-time" % "2.10.6"
   )
 
 val service = project
   .dependsOn(rules % ScalafixConfig)
   .settings(
-    scalaVersion := Versions.scala213,
-    libraryDependencies += "com.nequissimus" % "sort-imports_2.12" % "0.5.0" % ScalafixConfig
+    libraryDependencies += "com.nequissimus" %% "sort-imports" % "0.5.2" % ScalafixConfig
```

Comment:
https://github.com/NeQuissimus/sort-imports/pull/67

Reply:
By the way, I am using an external rule here as I need multiple rules, so I can't stick to `example-scalafix-rule`.